src/MetamorphosisServiceProvider.php

```diff
@@ -53,7 +53,7 @@ private function getGuzzleHttpClient(ConfigManager $configManager): GuzzleClient
         $options['base_uri'] = $configManager->get('url');
         $options['headers'] = array_merge(
             $this->getDefaultHeaders(),
-            $options['headers']
+            $options['headers'] ?? []
```

Comment:
is there a way to add a test for this?

Reply:
the sauce is more expensive than the chicken